openwisp_users/base/models.py

```diff
@@ -123,6 +123,21 @@ def organizations_dict(self):
         cache.set(cache_key, organizations, 86400 * 2)  # Cache for two days
         return organizations
 
+    def __get_orgs(self, attribute):
```

Comment:
Leading underscores methods are meant for internal usage only.
Double underscores is helpful to avoid in classes to avoid conflicts with subclasses, see https://www.python.org/dev/peps/pep-0008/#designing-for-inheritance:

> Always decide whether a class's methods and instance variables (collectively: "attributes") should be public or non-public. If in doubt, choose non-public; it's easier to make it public later than to make a public attribute non-public.
> 
> Public attributes are those that you expect unrelated clients of your class to use, with your commitment to avoid backwards incompatible changes. Non-public attributes are those that are not intended to be used by third parties; you make no guarantees that non-public attributes won't change or even be removed.
> 

...

> If your class is intended to be subclassed, and you have attributes that you do not want subclasses to use, consider naming them with double leading underscores and no trailing underscores. This invokes Python's name mangling algorithm, where the name of the class is mangled into the attribute name. This helps avoid attribute name collisions should subclasses inadvertently contain attributes with the same name.